added community fonts

spurin opened the pull request:
# Custom Fonts for Cilium, Tetragon & eBPF

As part of the recent efforts for the KubeCon Keynote Project Update, this PR includes the custom fonts and source files that were generated for use during the video editing process. These fonts were specifically created to enhance the visual identity of Cilium, Tetragon, and eBPF, allowing the logo's to be used natively within text based video editing workflows.

I hope these assets will be useful for future community projects and contributions.

spurin commented:
Assign @lizrice

spurin commented:
^^ Regarding the forced updates, we should now be in a finalised state. I noticed that FontForge, had automatically assigned Copyright of the fonts to myself using my system login id as a reference.

I've updated the Copyright section on each of the source files accordingly -

```
Copyright (c) 2025 The Cilium Authors

This Font Software is licensed under the SIL Open Font License, Version 1.1.  
You can obtain a copy at: https://scripts.sil.org/OFL
```

spurin commented:
Hi @xmulligan 

I'm no expert on fonts, ha :-) 

I searched and SIL Open Font License, Version 1.1 seemed to be the recommended choice. It is just a field in the meta data. We can however use anything we like.

If you drop in markup, what you'd prefer in the metadata, I'll quickly update the fonts and push another forced update to save on unnecessary commits (especially with these files being larger).

lizrice commented:
I had a little play with the fonts, they are brilliant! If the tool you're using allows you to drop in any license for the metadata, then the Apache 2.0 license would be ideal as it's the default for CNCF in general. I found [other](https://github.com/googlefonts/roboto-2/blob/main/LICENSE) [examples](https://online-fonts.com/fonts/droid-serif) of Apache 2.0 being used for fonts so we would be in good company

spurin commented:
Hi @lizrice and @xmulligan 

This should be now be good to go. I added the license metadata to the source fonts -

```
Copyright (c) 2025 The Cilium Authors

Licensed under the Apache License, Version 2.0 (the "License");  
you may not use this file except in compliance with the License.  
You may obtain a copy of the License at:

http://www.apache.org/licenses/LICENSE-2.0
```

And as a good measure, just in case this does make its way to the .ttf files, all of the ttf file have also been regenerated.

I've also added the LICENSE file for the Apache license to the folder.

xmulligan commented:
Thanks, really looking forward to all the ways we can use this in the future!
